_data/form.yml

```diff
@@ -0,0 +1,831 @@
+  - label: Current number of parking spaces
+    type: number
+
+  - label: Current number of bicycle spaces
+    type: number
+
+- title: What are you planning to change?
+  fields:
+  - label: Check all that apply.
+    type: checkbox
+    options:
+    - label: Parking gross square feet
+      shows: proposed-parking-gsf
+
+    - label: Residential gross square feet
+      shows: proposed-residential-gsf
+
+    - label: Retail commercial gross square feet
+      shows: proposed-commercial-gsf
+
+    - label: Number of affordable dwelling units
+      shows: proposed-affordable
+
+    - label: Number of market-rate dwelling units
+      shows: proposed-market-rate
+
+    - label: Number of buildings
+      shows: proposed-number-buildings
+
+    - label: Number of stories
+      shows: proposed-number-stories
+
+    - label: Number of basements or cellars
+      shows: proposed-number-basements
+
+    - label: Number of parking spaces
+      shows: proposed-parking-spaces
+
+    - label: Number of bicycle spaces
+      shows: proposed-bicycle-spaces
+
+- title: Fill in your proposed changes.
+  fields:
+  - group: proposed-parking-gsf
+    fields:
+    - label: Proposed parking space
+      type: number
+      unit: gross square feet
+
+  - group: proposed-residential-gsf
+    fields:
+    - label: Proposed residential space
+      type: number
+      unit: gross square feet
+
+  - group: proposed-commercial-gsf
+    fields:
+    - label: Proposed retail commercial space
+      type: number
+      unit: gross square feet
+
+  - group: proposed-affordable
+    fields:
+    - label: Proposed number of affordable dwelling units
+      type: number
+
+  - group: proposed-market-rate
+    fields:
+    - label: Proposed number of market-rate dwelling units
+      type: number
+
+  - group: proposed-number-buildings
+    fields:
+    - label: Proposed number of buildings
+      type: number
+
+  - group: proposed-number-stories
+    fields:
+    - label: Proposed number of stories
+      type: number
+
+  - group: proposed-number-basements
+    fields:
+    - label: Proposed number of basements or cellars
+      type: number
+
+  - group: proposed-parking-spaces
+    fields:
+    - label: Proposed parking spaces
+      type: number
+
+  - group: proposed-bicycle-spaces
+    fields:
+    - label: Proposed bicycle spaces
+      type: number
+
+- title: Fire protection
+  fields:
+  - label: Does the property have fire sprinklers?
+    type: radio
+    options:
+    - label: "Yes"
+      shows: sprinklers
+
+    - label: "No"
+      hides: sprinklers
+
+  - group: sprinklers
+    fields:
+    - label: Which areas of the property do the sprinklers cover?
+
+  - label: Does the property have fire alarms?
+    type: radio
+    options:
+    - label: "Yes"
+      shows: alarms
+
+    - label: "No"
+      hides: alarms
+
+  - group: alarms
+    fields:
+    - label: Which areas of the property do the alarms cover?
+
+- title: Worker's compensation insurance
```

Comment:
Scott: Will applicants have worker's compensation (or other prereqs) at the time of application?

For restaurants, many applicants didn't because of the long processing times. They'd have to pay 6 months of premiums while they waited